frontend/src/Editor/Components/Table/columns/index.jsx

```diff
@@ -94,7 +94,20 @@ export default function generateColumnsData({
       customRule: column?.customRule,
       sortType,
       columnVisibility: column?.columnVisibility ?? true,
-      Cell: function ({ cell, isEditable, newRowsChangeSet = null }) {
+      horizontalAlignment: column?.horizontalAlignment ?? 'left',
+      Cell: function ({ cell, isEditable, newRowsChangeSet = null, horizontalAlignment }) {
+        const determineJustifyContentValue = (value) => {
```

Comment:
The default case does not return anything explicitly, which means it implicitly returns undefined. It might be better to add a return null statement to the default case to ensure that any unexpected values return a defined value.